nautobot/ipam/tests/test_forms.py

```diff
@@ -25,14 +25,14 @@ def setUp(self):
             prefix="192.168.1.0/24", namespace=self.namespace, status=self.prefix_status
         )
         self.parent2 = Prefix.objects.create(
-            prefix="192.168.0.0/24", namespace=self.namespace, status=self.prefix_status
+            prefix="192.168.0.0/16", namespace=self.namespace, status=self.prefix_status
         )
         self.parent6 = Prefix.objects.create(
             prefix="2001:0db8::/40", namespace=self.namespace, status=self.prefix_status
         )
 
     def test_valid_ip_address(self):
-        data = {self.field_name: "192.168.1.0/24", "namespace": self.namespace, "status": self.status}
+        data = {self.field_name: "192.168.2.0/24", "namespace": self.namespace, "status": self.status}
         data.update(self.extra_data)
```

Comment:
What's the story on these changes?

Reply:
Because `PrefixFormTest` was still being skipped, we didn't catch the fact that `PrefixForm. test_valid_ip_address` was failing with `{'__all__': ['Prefix with this Namespace, Network and Prefix length already exists.']}` since that prefix is already being created as `self.parent1`. 

So this just expands it so that I didn't have to muck with other test code.